docs/transactions/iti41.md

```diff
@@ -1,35 +1,204 @@
+|```title```|yes|-|
+|```typeCode```|yes|See [section](#ClassCode-metadata-to-use-for-each-document-type) below.|
+|```uniqueId```|yes|Must be globally unique and be a UUID. Must match the document id in the case of CH-EMED-EPR documents.|
+|```URI```|no|-|
+|```version```|no|If present shall be empty, the value is ignored and set by the document registry.|
+|```DocumentEntry.patientId```|yes|Must match ```SubmissionSet.patientId``` and be a XAD-PID ([see above](#SubmissionSet-metadata)).|
+|```DocumentEntry.creationTime```|yes|Must match document content creation time ([see below](#Metadata-values-check)).|
+|```DocumentEntry.author```|yes|Must be aligned with document content author ([see below](#Metadata-values-check)).|
+
 
 ## Publishing a CH-EMED-EPR document
+This section details the specific rules to follow when publishing [CH-EMED-EPR documents](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/).
+
+ Only [MTP](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/document_mtp.html), [PRE](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/document_pre.html), [DIS](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/document_dis.html) and [PADV](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/document_padv.html) docs can be published. [PML](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/document_pml.html) and [PMLC](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/document_pmlc.html) can only be retrieved from the service. As a result, ```Iti41QueryConverter.process->EmedFhirSpec.getDocumentType``` returns ```null``` for ```PML``` and ```PMLC``` docs.
+
+### Referencing external documents
+When publishing CH-EMED-EPR documents, all references (from the key elements) shall be known to the eMedication service. All non-key elements are ignored by the eMedication service. 
+
+This implies that the metadata cannot contain [Reference ids](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.28). This option is not supported, and submission of a reference to an existing document is forbidden.
 
-When publishing a CH-EMED-EPR documents, all references (from the key elements) shall be known to the eMedication service.
-All non-key elements are ignored by the eMedication service.
+### Rules for each type of document
+#### MTP
+* Treatment duration (defined in [dosage](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage.html)) should be within the period of validity of the document (defined by [`DocumentEntry.serviceStatTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20) metadata).
+#### PRE
+* Effective time of [`documentationOf`](#FIXME) and [`serviceEvent`](#FIXME) has to be aligned with [`DocumentEntry.serviceStatTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20).
```

Comment:
I wasn't sure where to find the references to some of the document's attributes in the implementation guide. So I marked all of them with a #FIXME tag

Reply:
`documentationOf ` and `serviceEvent `are CDA, just remove them